Content.Server/Administration/Systems/AdminVerbSystem.cs

```diff
@@ -155,10 +155,9 @@ private void AddAdminVerbs(GetVerbsEvent<Verb> args)
                             var mobUid = _spawning.SpawnPlayerMob(coords.Value, null, profile, stationUid);
                             var targetMind = _mindSystem.GetMind(args.Target);
 
-                            if (targetMind != null)
-                            {
-                                _mindSystem.TransferTo(targetMind.Value, mobUid, true);
-                            }
+                            if (_mindSystem.TryGetMind(args.Target, out var mindId, out var _))
+                                _mindSystem.TransferTo(mindId, mobUid, true);
```

Comment:
```suggestion
                            if (_mindSystem.TryGetMind(args.Target, out var mindId, out var mindComp))
                                _mindSystem.TransferTo(mindId, mobUid, true, mind: mindComp);
```
you can reuse the component instead of discarding it